### PR TITLE
feature: Add versioned feeds file [prod]

### PR DESCRIPTION
**Pull request by ramos-ph**
Adds Options and Yearn feeds to Prod BSC.

**Note:** The `feeds-5.5.yaml` file is temporary and shall be removed in the future. We had to create it just for pegasus version 5.5.0. That way, we avoid older versions from breaking.

### CODE REVIEW

File: prod/bsc/feeds.yaml
Comment (dougmrqs) [inline prod/bsc/feeds.yaml:9919]: gotta align these OP to match the new asterisk and calculator stuff.

**Review by dougmrqs — CHANGES_REQUESTED** [review]

File: prod/bsc/feeds-5.5.0.yaml
Comment (dougmrqs) [inline prod/bsc/feeds-5.5.0.yaml:9900]: Will we need this for prod? Please confirm with AV. Does this post ropsten stuff on mainnet?

File: prod/bsc/feeds.yaml
Comment (ramos-ph) [reply prod/bsc/feeds.yaml:9919]: Leave it to me

**Review by dougmrqs — APPROVED** [review]
Make sure you squash these 5 commits before merging the PR. Or rebase them to be more concise with the feature.

**Review by dougmrqs — CHANGES_REQUESTED** [review]
Let's squash these 9 commits into something more concise. We have 2 files, two changes, and 9 commits.

**Comment by ramos-ph** [comment]
> Let's squash these 9 commits into something more concise. We have 2 files, two changes, and 9 commits.

I rebased it locally and reduced it to 2 commits. I'd like you to give it a look before force pushing it.

**Review by dougmrqs — APPROVED** [review]

**Pull request merged**
